### PR TITLE
[SFX-25] Expose port 9000 for php-fpm

### PR DESCRIPTION
**Pull request by damacus**

### CODE REVIEW

File: opg-php-fpm-71-ppa-1604/Dockerfile [hunk 1]
```diff
@@ -3,56 +3,58 @@ FROM registry.service.opg.digital/opg-nginx-1604
...
-    apt-get clean && apt-get -y autoremove &&\
-    update-alternatives --set php /usr/bin/php7.1 &&\
-    echo "Making sure scripts are executable" &&\
-    chmod a+x /etc/my_init.d/* &&\
-    mkdir -p /data/session
+RUN echo "Adding PHP7.1 PPA" \
+  && chmod 1777 /tmp/ \
+  && add-apt-repository -y ppa:ondrej/php \
+  && apt-get update -y \
+  && apt-get install -y sudo \
```
Comment (webit4me) [inline opg-php-fpm-71-ppa-1604/Dockerfile]: I think you meant to drop `sudo` in the next line too :)

Comment (webit4me) [inline opg-php-fpm-71-ppa-1604/Dockerfile]:
```suggestion
  && apt-get install -y \
```

File: opg-php-fpm-71-ppa-1604/Dockerfile [hunk 2]
```diff
@@ -3,56 +3,58 @@ FROM registry.service.opg.digital/opg-nginx-1604
...
+  python-software-properties \
```
Comment (webit4me) [inline opg-php-fpm-71-ppa-1604/Dockerfile]:
```suggestion
  sudo python-software-properties \
```

File: opg-php-fpm-71-ppa-1604/Dockerfile [hunk 3]
```diff
@@ -3,56 +3,58 @@ FROM registry.service.opg.digital/opg-nginx-1604
...
+  && phpdismod -v 7.1 xdebug \
+  && echo "xdebug.remote_enable=on\nxdebug.remote_port=9000\nxdebug.remote_host=10.254.254.254" \
+  >> /etc/php/7.1/mods-available/xdebug.ini \
+  && echo "Replacing php-fpm configuration" \
+  && rm -f /etc/php/7.1/fpm/php-fpm.conf /etc/php/7.1fpm/pool.d/www.conf /etc/cron.d/php \
+  && chmod a+x /etc/sv/php-fpm/run \
+  && ln -s /etc/sv/php-fpm /etc/service/ \
+  && echo "Installing Composer" \
+  && chmod a+x /tmp/install-composer.sh \
+  && . /tmp/install-composer.sh \
```
Comment (webit4me) [inline opg-php-fpm-71-ppa-1604/Dockerfile]:
```suggestion
  && source /tmp/install-composer.sh \
```

Comment (damacus) [reply opg-php-fpm-71-ppa-1604/Dockerfile]: This sadly doesn't work :(

Comment (webit4me) [reply opg-php-fpm-71-ppa-1604/Dockerfile]: I've added this suggestion to keep it consistent with rest of your improvements, but chance we need to keep them all `.` as `source` might not be available on the agent

**Pull request closed without merging**